Bonsai.Emergent/Bonsai.Emergent.csproj

```
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Title>Bonsai.Emergent</Title>
+    <Description>A package for the Bonsai visual programming language.</Description>
+    <Authors></Authors>
+    <Copyright>Copyright ©  2024</Copyright>
+    <GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">true</GeneratePackageOnBuild>
+    <PackageProjectUrl></PackageProjectUrl>
+    <PackageLicenseExpression></PackageLicenseExpression>
+    <PackageIcon></PackageIcon>
+    <PackageOutputPath></PackageOutputPath>
+    <PackageTags>Bonsai Rx Bonsai.Emergent</PackageTags>
+    <TargetFramework>net481</TargetFramework>
+    <Features>strict</Features>
+    <Version>0.1.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Bonsai.Core" Version="2.8.2" />
+    <PackageReference Include="OpenCV.Net" Version="3.4.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="EmergentCameraDotNet">
+      <HintPath>..\..\..\..\..\..\..\Program Files\EVT\eSDK\bin\EmergentCameraDotNet.dll</HintPath>
```

```suggestion
      <HintPath>$(EMERGENT_DIR)\eSDK\bin\EmergentCameraDotNet.dll</HintPath>
```

I noticed the eSDK seems to configure an environment variable pointing to the EVT folder, so it might be best to use it to avoid issues with local folder structure.

We can think of other ways to improve this later. Also need to think of ways to build the package on CI/CD, which might not be possible without having a copy of the DLL in the repo, but we can worry about that later.